arcgis-ios-sdk-samples/Layers/Display subtype feature layer/README.metadata.json

```
@@ -0,0 +1,26 @@
+{
+  "category": "Features",
+  "description": "Display a composite layer of all the subtype values in a feature class.",
+  "ignore": false,
+  "images": [
+    "Display-subtype-feature-laer.png"
```

```suggestion
    "Display-subtype-feature-layer.png"
```
There is a typo - please double check if my change is correct.